docs/Makefile

```diff
@@ -15,7 +15,7 @@ clean:
 	find $(SOURCEDIR) -name "*.rst" | grep -v index | xargs rm -f
 
 apidoc:
-	sphinx-apidoc -f --templatedir=$(TEMPLATEDIR) -o $(SOURCEDIR) ../datasetinsights/ ../datasetinsights/visualization ../datasetinsights/dashboard.py ../datasetinsights/data/datasets/protos
+	sphinx-apidoc -f --templatedir=$(TEMPLATEDIR) -o $(SOURCEDIR) ../datasetinsights/ ../datasetinsights/ststs/visualization ../datasetinsights/dashboard.py ../datasetinsights/datasets/protos
 html: apidoc
```

Comment:
datasetinsights/ststs/visualization: type in ststs

Reply:
Thanks for catching this. It's been fixed now.